setup.py

```diff
@@ -34,13 +29,17 @@
   long_description_content_type='text/markdown',
   url='https://github.com/google/shaka-streamer',
   packages=setuptools.find_packages(),
-  install_requires=install_prerequisites,
+  install_requires=[
+      'PyYAML',
+      'pywin32;platform_system=="Windows"',
```

Comment:
I just found out that we don't need `pypiwin32` and just `pywin32` would be enough.